components/src/core/list-item-tag/list-item-tag.component.html

```diff
@@ -0,0 +1,9 @@
+<div class="pxb-list-item-tag-wrapper" 
+    
+    [style.backgroundColor]="backgroundColor">
```

Comment:
are we doing background and font color through attributes, or should we be leaving this to the user to customize through classes?

Reply:
I think the bg/font color @inputs are fine in this case because the component is so small.   The component should be able to support styles through classes here if there is no @input colors provided.  This'll have to be true for the theme to work.

Reply:
I dont have any issues removing the font/bg color inputs for the sake of consistency.

Reply:
I like the idea of consistency, but I fear that if we take them out, then the component doesn't really do anything. I'm okay with keeping them for the time being.

Reply:
Sounds good to me, I think the component can exist without the input/bg color inputs if we do decide to remove them in the future.  It'll still have a few things going for it - PXBTheme, typography & padding so it looks great in our InfoListItem.